### DISCUSSION TITLE
【八卦】新艾利都治安局的朱鸢警长去海边了，有照片为证！

### DISCUSSION THREAD

**Discussion by Manbow1997** [share121/inter-knot#3107] [其他]
![c76cef026fb9e948906dedf4390344a4](https://github.com/user-attachments/assets/d73c47d1-f2d9-4bfe-8f9e-96f925ce2835)
@ibaraki_shun
今天我去海边，远远就看到治安局的人也在，一个小个子，就那个社区访问的青衣警官，还有朱鸢警长，还穿的泳装，我的天呐，朱鸢警长的身材也太好了吧，我鼻血都要止不住了。前段时间，一个叫山什么猫的团伙，听说很狡猾，都被朱鸢警长重拳出击了。我也好想被朱鸢警长奖励啊！不过有只鲨鱼布在那，我不敢靠近，怕被咬，所以就在远处偷偷拍了一张。吃鱼的，小小的嘿嘿；还有俩钓鱼的，前面这个也太太...太大了吧，我的天，后面那个还有打伞的白毛，这些人有知道的吗？这是在执行什么任务吗？也不像呀。

**Comment by WWL354459** [comment]
好好好，多写爱看。

**Comment by Nie7bai** [comment]
喜欢看捏<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 96u** [comment]
小猫咪嘿嘿嘿小猫咪
